smarkets/streaming_api/session.py

```diff
@@ -132,6 +134,8 @@ def logout(self):
     def disconnect(self):
         "Disconnects from the API"
         self.socket.disconnect()
+        self.inseq = self.init_inseq
+        self.outseq = self.init_outseq
```

Comment:
@sandrovicente Why is this required?

Reply:
inseq and outseq may be given as init parameters. I save the ones received to 'init_XXX' so in case of reconnection I keep using them. Not really sure of a use case for this though, only to keep consistent